src/modules/ROOT/pages/changelog/202303/00.json

```diff
@@ -0,0 +1,808 @@
+      "why": [
+        "to extend record filtering support for range filtering of numeric and date-time for single value field."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/813/commits/2027409b0d8875a48fd0017264ba03f5535733ec"
+      ]
+    },
+    {
+      "what": [
+        "new support for a fully customizable navigation page block with multiple navigation item type and customizable options for the navigation list items."
+      ],
+      "why": [
+        "to have the ability to show one or two level of dynamic and static navigation."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/41f1283e9d393393c651b8ac5a28cae09733f1fd"
+      ]
+    },
+    {
+      "what": [
+        "ability to drill down chart, by clicking on a data entry inside on the chart.",
+        "Based on the configuration it can either filter an existing record list on the page, that is showing the records from the same module as the chart, or open a new record list in a modal."
+      ],
+      "why": [
+        "to make it easier for users to filter record lists based on chart data, in order to view the details of said data."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/877/commits/cfc4ab7424ed1a131bb34cfaed8c4ca18eef1b07"
+      ]
+    },
+    {
+      "what": [
+        "support for conditional fields and being able to be displayed based on evaluated expressions."
+      ],
+      "why": [
+        "to give users more control on how records are displayed."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/770/commits/1d286aff83f012e33cabc3456ffa175f7c105eb7"
+      ]
+    },
+    {
+      "what": [
+        "support to allow users to navigate to previous and next record page."
+      ],
+      "why": [
+        "to allow users to navigate to previous and next record page."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/836/commits/ce79e8233d40894657c00e806fa50589c2724c92"
+      ]
+    },
+    {
+      "what": [
+        "support for bulk record updates."
+      ],
+      "why": [
+        "to allow the users to update multiple records in a record list."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/906/commits/97a9cd877ab11917dac9f031db36cbd8f4d215b3",
+        "https://github.com/cortezaproject/corteza/pull/906/commits/717e95b22efefc329198dd4689b0321273813cf8",
+        "https://github.com/cortezaproject/corteza/pull/906/commits/f8a3da0bdf40e010fde9902cc2954a6d5973b75d",
+        "https://github.com/cortezaproject/corteza/pull/906/commits/656bea5636f690cdf2697349695011734f24bf49",
+        "https://github.com/cortezaproject/corteza/pull/906/commits/508e8bab92d6eb2ca6fce17c807d4a33d79ae9ad"
+      ]
+    },
+    {
+      "what": [
+        "support for a new page block in {APP_NAME_COMPOSE} called tabs block where each tab is a different block."
```

Comment:
I'd leave this as a separate entry since I think it is a good addition to our platform

Reply:
The point I was trying to make is that I think it'd be better if these similar bits (new page blocks, new page builder options, new ____) would be physically together. Why is a one-page block higher up with some _random_ feature in the middle and then this? I'd expect them together.

I'm ok with having separate entries; but if we do, let's have them together and written in a similar style.

What I'd do (It would be possible with our _framework_ also) is:

[Added ]new page blocks to do things and bits and stuff:
* **Tabs** which does this and that,
* **Navigation** which does more then this and that.

If you opt-in for my proposal, you'll need to have it like so:

```json
{
  "added": [{
    "what": [
        "new page blocks to do things and bits and stuff:",
        "* **Tabs** which does this and that,",
        "* **Navigation** which does more then this and that."
    ]
  }]
}
```

Re-consider and adjust if you see it fit